sgx_ci/Dockerfile.sgx

```diff
@@ -0,0 +1,125 @@
+FROM ubuntu:22.04 as minimal-sgx
```

Comment:
What about create a `ci` folder and put another `sgx` inside then put all the contents you created inside.
Just a reorganisation idea.